src/EnliteMonolog/Service/MonologServiceFactory.php

```diff
@@ -57,6 +58,10 @@ public function createLogger($container, MonologOptions $options)
             $logger->pushProcessor($this->createProcessor($container, $processor));
         }
 
+        if (class_exists('\Monolog\ErrorHandler') && $options->isErrorHandler()) {
```

Comment:
Which version of php are we support? If minimum 5.4 then you can use `::class` here.